functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -58,6 +59,21 @@ def wait_for_scylla_operator_rollout_complete(db_cluster: ScyllaPodCluster, time
     return status
 
 
+def wait_for_resource_absence(db_cluster: ScyllaPodCluster,
+                              resource_type: str, resource_name: str,
+                              step: int = 2, timeout: int = 60):
+    def resource_is_absent(resource_type, resource_name: str):
```

Comment:
Consider to not pass any values to this function because you define it as a nested function. And, please, add a type hint for the return value (`bool`?)

Reply:
Done